CHANGELOG

```diff
@@ -3,6 +3,7 @@
   - Replace unicorn with puma in production
   - Add importmap-rails to handle js libraries
   - Revision history: Improve version history for content with carriage return
+  - Activity filtering
```

Comment:
This doesn't meet the defined changelog format.

```[entity]: [future tense verb] [feature]```

To be fair a few other entires here don't either. We'll have to update those before the release, but let's get this one sorted out here.